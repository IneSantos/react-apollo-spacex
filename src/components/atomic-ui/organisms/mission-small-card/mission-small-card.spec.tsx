import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";
import MissionSmallCard from "./mission-small-card";

describe("MissionSmallCard organism tests", () => {
    test(`MissionSmallCard component render`, () => {
        const mission = {
            id: "0",
            name: "This is a mission name",
            details: "This is a mission detail",
            date: "2022/04/10"
        }
        const { container } = render(
            <Router>
                  <MissionSmallCard {...mission}/>
            </Router>
        );
        expect(container.firstChild).toMatchSnapshot();
    });
});
