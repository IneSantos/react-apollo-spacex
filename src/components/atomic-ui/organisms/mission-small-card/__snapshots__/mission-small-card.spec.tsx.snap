// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MissionSmallCard organism tests MissionSmallCard component render 1`] = `
<a
  href="/mission/0"
  style="text-decoration: none;"
>
  <div
    class="mission-wrapper"
  >
    <div
      class="card-header"
    >
      <h2
        class="title"
      >
        This is a mission name
      </h2>
    </div>
    <div
      class="list-card-body"
    >
      <div
        class="details"
      >
        This is a mission detail
      </div>
      <div
        class="launch-date"
      >
        April 10, 2022
      </div>
    </div>
  </div>
</a>
`;
